test_accounts/health_test_accounts.md

```diff
@@ -1,6 +1,6 @@
 # Sandbox Test Accounts for Health APIs
 
-**Veteran Health** test users have a variety of synthetic health records attached to them and can be used to test the [Veteran Health API](https://developer.va.gov/explore/health/docs/fhir?version=current). 
+**Veteran Health** test users have a variety of synthetic health records attached to them and can be used to test the [Veteran Health API](https://developer.va.gov/explore/health/docs/fhir?version=current). A small subset of these can also be used for testing the [Clinical Health API](https://developer.va.gov/explore/health/docs/clinical_health?version=current)
```

Comment:
1) I would change "A small subset of these can also be..." to "A small subset of these users can also be..." - add the word users after these.

2) I'd put a period at the end of this sentence - "A small subset of these users can also be used for testing the [Clinical Health API](https://developer.va.gov/explore/health/docs/clinical_health?version=current)."

Reply:
Updated as specified